docs/ref/modules/vulnerability-scanner/events.md

```diff
@@ -0,0 +1,670 @@
+# Events format 
+
+The incoming events from the agents must be parseable by the [flatbufer schemas](flatbuffers.md), otherwise it will trigger an exception and the event won't reach the vulnerability scanner. Below we detail the format for the different systems supported by the scanner.
```

Comment:
```suggestion
The incoming events from the agents must be parseable by the [flatbuffer schemas](flatbuffers.md), otherwise it will trigger an exception, and the event won't reach the vulnerability scanner. Below, we detail the format for the different systems supported by the scanner.
```